### PR TITLE
Extended transaction monitor

### PR DESCRIPTION
**Pull request by alexanderad**
* transaction monitor now supports `account` param
* proper `get_min_ledger_index` implementation

### CODE REVIEW

**Comment by alexanderad** [comment]
For merge after integration to `votevest`.

File: ripple_api/management/transaction_processors.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    """
+    if transaction or args:
+        format_args = [transaction]
+        format_args.extend(args)
+        return message % tuple(format_args)
+    else:
+        return message
+
+
+def monitor_transactions(account, logger=DEFAULT_LOGGER):
```
Comment (42) [inline ripple_api/management/transaction_processors.py]: It would be worth refactoring this into several shorter functions?

Comment (alexanderad) [inline ripple_api/management/transaction_processors.py]: image

Comment (42) [inline ripple_api/management/transaction_processors.py]: (y)

**Comment by alexanderad** [comment]
@42 ready for review

**Pull request merged**